README.md

```diff
@@ -441,6 +441,27 @@ var p = new Proxy(target, handler);
 p() === 'I am the proxy';
 ```
 
+```JavaScript
+// Advanced field interception with Proxy and the Reflect object
+// Reflect is a built-in object that provides methods for interceptable
+// JavaScript operation. The methods are the same as those of proxy
+// handlers. Reflect is not a function object, so it's not constructable.
+
+var advance = new Proxy({}, {
```

Comment:
`advanced` I think?

Reply:
Good catch @hlomzik 👍

Reply:
See https://github.com/lukehoban/es6features/pull/129/commits/b559ef647e8747faf267a5e29b3ac7e50c9aea56 for changes